rustls/Cargo.toml

```diff
@@ -30,6 +59,10 @@ std = ["webpki/std", "pki-types/std", "once_cell/std"]
 tls12 = []
 zlib = ["dep:zlib-rs"]
 
+# FEATURES to help build for no-std targets with no atomic ptr - see documentation for more info
+critical-section = ["dep:portable-atomic", "once_cell/portable-atomic", "portable-atomic/critical-section"]
+unsafe-assume-single-core = ["once_cell/portable-atomic", "portable-atomic/unsafe-assume-single-core"]
```

Comment:
I think "dep:portable-atomic" should have been part of `unsafe-assume-single-core` entry for better clarity:

```diff
-unsafe-assume-single-core = ["once_cell/portable-atomic", "portable-atomic/unsafe-assume-single-core"]
+unsafe-assume-single-core = ["dep:portable-atomic", "once_cell/portable-atomic", "portable-atomic/unsafe-assume-single-core"]
```

(This will likely change in upcoming PR #49 or #44 anyways.)